plugins/obs-ffmpeg/jim-nvenc.c

```diff
@@ -949,8 +949,7 @@ static bool init_textures(struct nvenc_data *enc)
 static void nvenc_destroy(void *data);
 
 static bool init_specific_encoder(struct nvenc_data *enc, bool hevc,
-				  obs_data_t *settings, obs_encoder_t *encoder,
-				  int bf, bool psycho_aq)
+				  obs_data_t *settings, int bf, bool psycho_aq)
 {
 	bool init = false;
```

Comment:
I mean this variable. I didn't even see the other one, lol. Good to get rid of both.

Reply:
yea, i got it.
I just thought the variable was reserved for something by you when I was fixing.

Reply:
You removed `bool init = false;`? I still see it.

Reply:
no. I re-push it now.